cmd/kubectl-gadget/version.go

```diff
@@ -15,16 +15,29 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 
-	"github.com/spf13/cobra"
-
 	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
 	"github.com/inspektor-gadget/inspektor-gadget/internal/version"
 	grpcruntime "github.com/inspektor-gadget/inspektor-gadget/pkg/runtime/grpc"
+	"github.com/spf13/cobra"
 )
 
+// VersionInfo represents the structure for version information output
+type VersionInfo struct {
+	ClientVersion *Version `json:"clientVersion,omitempty"`
+	ServerVersion *Version `json:"serverVersion,omitempty"`
+}
+
+// Version contains detailed version information
+type Version struct {
+	Version string `json:"version"`
+	Status  string `json:"status,omitempty"`
```

Comment:
I'm not sure about this status field. I think we don't need it. (see comment below)